src/scenes/levelend-scene.ts

```diff
@@ -147,7 +148,10 @@ export class LevelEndScene {
       this.monster.update(deltaTime);
       this.closeButton.draw();
       this.retryButton.draw();
-      if (this.starCount >= 2) {
+      if (
+        this.starCount >= 2 &&
+        this.isLastLevel
```

Comment:
do we need both condition for this condition or this.isLastLevel would suffice?

Reply:
We need both, because if we remove starCount, it means the user can go to the next level even if they only got 1 or 0 star. We need 2 stars in order to move to the next level.

Reply:
I am sure about this. The task is to remove the next level on level-end scene if it is the last game level.

Reply:
there is nothing else to check other then the last level, may it be 0 star 1 star or 3 star there is an option to replay always but there should never be a next level button if it is the last level of the game. we can remove the starcount check from the condition.